src/js/models/appUser.js

```diff
@@ -42,7 +42,7 @@ var AppUser = module.exports = BaseModel.extend({
             return Backbone.Model.prototype.save.apply(this, _.toArray(arguments)).then(function() {
                 this._lastPropertyValues = this.pick(AppUser.EDITABLE_PROPERTIES);
                 return arguments;
-            });
+            }.bind(this));
```

Comment:
We're not supporting old crumbling browsers though...
I hate the way _.bind looks.
If we have to go through the codebase to fix them all, I volunteer.